ext/civicrm_admin_ui/managed/SavedSearch_Print_Page_PDF_Formats.mgd.php

```diff
@@ -0,0 +1,188 @@
+          'version' => 4,
+          'select' => [
+            'id',
+            'label',
+            'description',
+            'is_active',
+            'is_default',
+          ],
+          'orderBy' => [],
+          'where' => [
+            [
+              'option_group_id:name',
+              '=',
+              'pdf_format',
+            ],
+          ],
+          'groupBy' => [],
+          'join' => [],
+          'having' => [],
+        ],
+      ],
+      'match' => [
+        'name',
+      ],
+    ],
+  ],
+  [
+    'name' => 'SavedSearch_Print_Page_PDF_Formats_SearchDisplay_Option_Values_Table_1',
+    'entity' => 'SearchDisplay',
+    'cleanup' => 'unused',
+    'update' => 'unmodified',
+    'params' => [
+      'version' => 4,
+      'values' => [
+        'name' => 'Option_Values_Table_1',
+        'label' => E::ts('Option Values'),
+        'saved_search_id.name' => 'Print_Page_PDF_Formats',
+        'type' => 'table',
+        'settings' => [
+          'description' => NULL,
+          'sort' => [],
+          'limit' => 50,
+          'pager' => [
+            'show_count' => FALSE,
+            'expose_limit' => FALSE,
+          ],
+          'placeholder' => 5,
+          'columns' => [
+            [
+              'type' => 'field',
+              'key' => 'label',
+              'dataType' => 'String',
+              'label' => E::ts('Name'),
+              'sortable' => TRUE,
+            ],
+            [
+              'type' => 'field',
+              'key' => 'description',
+              'dataType' => 'Text',
+              'label' => E::ts('Description'),
+              'sortable' => TRUE,
+            ],
+            [
+              'type' => 'field',
+              'key' => 'is_default',
+              'dataType' => 'Boolean',
+              'label' => E::ts('Default'),
+              'sortable' => TRUE,
+            ],
+            [
+              'size' => 'btn-xs',
+              'links' => [
+                [
+                  'entity' => '',
+                  'action' => '',
+                  'join' => '',
+                  'target' => 'crm-popup',
+                  'icon' => 'fa-external-link',
+                  'text' => E::ts('Edit'),
+                  'style' => 'default',
+                  'path' => 'civicrm/admin/pdfFormats?action=update&id=[id]&reset=1',
+                  'task' => '',
+                  'condition' => [],
+                ],
+                [
+                  'entity' => 'OptionValue',
+                  'action' => 'delete',
+                  'join' => '',
+                  'target' => 'crm-popup',
+                  'icon' => 'fa-trash',
+                  'text' => E::ts('Delete'),
+                  'style' => 'danger',
+                  'path' => '',
+                  'task' => '',
+                  'condition' => [],
+                ],
+              ],
+              'type' => 'buttons',
+              'alignment' => 'text-right',
```

Comment:
This split with some links in a "buttons" group and others in a "menu" group isn't typical. The convention in Admin UI is to place all links in a single "menu" group.